python/lsst/ts/vent/controller/controller.py

```diff
@@ -327,3 +403,41 @@ def vent_state(self, vent_number: int) -> VentGateState:
                 return VentGateState.CLOSED
             case _:
                 return VentGateState.FAULT
+
+    def getOptoCh(self, *args, **kwargs) -> int:
```

Comment:
Method should be `snake_case`. But we need to talk about the use of this `megaind` dependency.